methods_and_recursion/mad_libs.rb

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env ruby
+
+def give_me(request)
+  print "#{request} "
+  gets.chomp
+end
+
+adj1 = give_me('Give me an adjective: ')
+adj2 = give_me('Give me another adjective: ')
+noun1 = give_me('Give me a noun: ')
+noun2 = give_me('Give me another noun: ')
+plural_noun = give_me('Give me plural noun: ')
+game = give_me('Give me a game: ')
+plural_noun2 = give_me('Give another plural noun: ')
+verb1 = give_me("Give me a verb ending in 'ing': ")
+verb2 = give_me("Give me another verb ending in 'ing': ")
+plural_noun3 = give_me('Give me another plural noun: ')
+verb3 = give_me("Give another verb ending in 'ing': ")
+noun3 = give_me('Give me another noun: ')
+plant = give_me('Give me a plant: ')
+body_part = give_me('Give me a body part: ')
+place = give_me('Give me a place: ')
+verb4 = give_me("Give me another verb ending in 'ing': ")
+adj3 = give_me('Give me another adjective: ')
+number = give_me('Give me a number: ')
+plural_noun4 = give_me('Give me another plural noun: ')
+
+puts 'Here is your madlib:'
```

Comment:
Consider a heredoc...  http://makandracards.com/makandra/1675-using-heredoc-for-prettier-ruby-code